src/java.base/share/classes/java/util/ReverseOrderListView.java

```diff
@@ -395,6 +395,11 @@ public List<E> subList(int fromIndex, int toIndex) {
         return new ReverseOrderListView<>(base.subList(size - toIndex, size - fromIndex), modifiable);
     }
 
+    @Override
```

Comment:
I don't use the Override annotation in collections. It can be useful for catching errors, but once the code is tested properly, having the annotation merely adds clutter.